lib/t/feeds.ex

```diff
@@ -46,6 +48,7 @@ defmodule T.Feeds do
           String.t() | nil
         ) ::
           {[%FeedProfile{}], String.t()}
+  # TODO remove writes
```

Comment:
@akhrail1996 can writes to DB be removed from `fetch_feed`? Then it would benefit from read replicas.

Reply:
@ruslandoga I think so, since these profiled should be filtered out by this: https://github.com/getsince/test3/blob/master/lib/t/feeds.ex#L250

gotta test though, and maybe adjust something on frontend as well

Reply:
@ruslandoga actually, no, my bad
In any case batch fetching happens before every profile is seen => we can not just remove this. Maybe store them in memory?

Reply:
@akhrail1996 can a cursor be used instead of feeded_profiles?

Reply:
why distance?

Reply:
wouldn't there be a language barrier?

Reply:
No, I don't think there should be artificial barriers. Since communication happens outside the language barrier is also not a big problem.
Currently I am using postgis for distance-based sorting, and it's working well. But I am open to switching to other solutions if you switch to sqlite.

Reply:
sure: https://github.com/getsince/test3/pull/637
beware, it's ugly 🙃

Reply:
feed fetching optimisation would happen in some next pr